ycsb-rs/src/main.rs

```diff
@@ -64,9 +73,11 @@ async fn run_ob(
     db: Arc<OBKVClient>,
     rng: Arc<Mutex<SmallRng>>,
     operation_count: usize,
+    counter: Arc<AtomicUsize>,
 ) {
     for _ in 0..operation_count {
         wl.ob_transaction(rng.clone(), db.clone()).await;
+        counter.fetch_add(1, Ordering::SeqCst);
```

Comment:
If just use it as a counter, order can be set to `Ordering::Relaxed`.

Reply:
Fixed